Tests/EndToEnd.tests.ts

```diff
@@ -70,8 +75,11 @@ class fakeRequest {
     public write(): void { }
     public headers: { [id: string]: string } = {};
     public agent = { protocol: 'http' };
+    private _responseData: any;
```

Comment:
Add this variable so when creating a fakeRequest, I can specify what response data I'll receive. So when it creates a faceResponse instance on line 108, it passes in this responseData.